sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/RemoteDependencyDataNewTests.cs

```diff
@@ -28,7 +28,7 @@ public void DependencyTypeIsSetToInProcForInternalSpan()
             Assert.NotNull(activity);
             var activityTagsProcessor = TraceHelper.EnumerateActivityTags(activity);
 
-            var remoteDependencyDataType = new RemoteDependencyData(2, activity, ref activityTagsProcessor, schemaVersion:SchemaConstants.DefaultSchemaVersion).Type;
+            var remoteDependencyDataType = new RemoteDependencyData(2, activity, ref activityTagsProcessor).Type;
```

Comment:
New and Old test is missing validation on target and resultcode. Need to add these fields.

Reply:
Fixed.